examples/window/scale_factor_override.rs

```diff
@@ -63,20 +63,39 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
         });
 }
 
+/// Set the title of the window to the current override
+fn display_override(mut windows: Query<&mut Window>) {
+    let mut window = windows.single_mut();
+
+    window.title = format!(
+        "Scale override: {:?}",
+        window.resolution.scale_factor_override()
+    );
+}
+
 /// This system toggles scale factor overrides when enter is pressed
-fn toggle_override(input: Res<Input<KeyCode>>, mut windows: ResMut<Windows>) {
-    let window = windows.primary_mut();
+fn toggle_override(input: Res<Input<KeyCode>>, mut windows: Query<&mut Window>) {
+    let mut window = windows.single_mut();
+
     if input.just_pressed(KeyCode::Return) {
-        window.set_scale_factor_override(window.scale_factor_override().xor(Some(1.)));
+        let scale_factor_override = window.resolution.scale_factor_override();
+        window
+            .resolution
+            .set_scale_factor_override(scale_factor_override.xor(Some(1.0)));
     }
 }
 
 /// This system changes the scale factor override when up or down is pressed
-fn change_scale_factor(input: Res<Input<KeyCode>>, mut windows: ResMut<Windows>) {
-    let window = windows.primary_mut();
+fn change_scale_factor(input: Res<Input<KeyCode>>, mut windows: Query<&mut Window>) {
```

Comment:
Setting the scale factor override in this example still doesn't actually set the override.

Reply:
I think this should be fixed now, just misunderstood what the scaling factor did before

Reply:
This bug is fixed!